projects/packages/search/src/class-options.php

```diff
@@ -46,7 +46,6 @@ class Options {
 	 * @var string
 	 */
 	const OVERLAY_TRIGGER_IMMEDIATE = 'immediate';
-	const OVERLAY_TRIGGER_RESULTS   = 'results';
```

Comment:
Worried that removing this might cause deploy-time fatals on wpcom 🤔 Could just mark as deprecated?

Reply:
Good point - it seems `wp-content/mu-plugins/site-search/class-jetpack-search-customize.php:L143` is referencing the constant.